pinot-core/src/test/java/com/linkedin/pinot/index/readerwriter/FixedByteSingleColumnMultiValueReaderWriterTest.java

```diff
@@ -62,4 +73,62 @@ public void testIntArray(final long seed)
     }
     readerWriter.close();
   }
+
+  public void testIntArrayFixedSize(int multiValuesPerRow, long seed)
+      throws IOException {
+    FixedByteSingleColumnMultiValueReaderWriter readerWriter;
+    int rows = 1000;
```

Comment:
rows  = random(1, multiValuesPerRow) to test chunking. I think the caller goes up to multiValuesPerRow of 1000 whcih may not test chunking